### ISSUE TITLE
Trouble importing lancedb with fiftyone

### ISSUE THREAD

**Issue by Deere-davinmiller** [lancedb/lancedb#387]
**Description:**
I'm having trouble importing fiftyone after pip installing lancedb. It appears that the attr package is conflicting with the attrs package:
```
`raise NotImplementedError  # pragma: no cover
---> 86 @attr.s(auto_exc=True, auto_attribs=True)
     87 class Error(Exception):
     88     """
     89     Base error class.
     90     """
     92     message: str

AttributeError: module 'attr' has no attribute 's'`
```

I've noticed that installing lancedb installs the attr package, which may be what's causing the issue for me.
I've installed the newest version of fiftyone (0.21.4) and it imports fine as long as I don't install lancedb beforehand.
Has anyone else had this issue?

**Comment by AyushExel** [comment]
Did this work fine for previous version of fiftyone?

**Comment by Deere-davinmiller** [comment]
No, I had originally tried with an older version (0.20.*) but it had the same issue. I reduced my requirements.txt to simply: 
```
fiftyone
lancedb
```

and I still cannot import fiftyone without error. I am running on Databricks runtime 12.2 ML which I'm guessing is causing some compatibility issues. (https://docs.gcp.databricks.com/release-notes/runtime/12.2lts-ml.html)

**Comment by AyushExel** [comment]
@Deere-davinmiller we'll work on a permanent fix for this. In the meantime, can you try manually installing the attr version compatible with fiftyone as the last step? let us know if that fixes it for you. thanks!

**Comment by Deere-davinmiller** [comment]
I got it to work by upgrading my attrs package from 21.4.0 to 23.1.0, so I guess the older attrs package from runtime 12.2ML couldn't work with the current packages in lancedb. 

Another package issue was with numpy, as lancedb installs the newest version of numpy, which is incompatible with parts of fiftyone. Specifically, when the "numba" package is used it requires numpy version <= 1.21. From the error message, I believe this occurs when calling compute_visualization() with the default umap dimensionality reduction.

**Comment by ericfeunekes** [comment]
I'm getting a similar issue with Databricks Runtime 13.3 LTS. I need to pin the version of `attrs`:

```python
%pip install lancedb 'attrs>=23.0.0'
```

Then restart the kernel

```python
dbutils.library.restartPython()
```

**Comment by changhiskhan** [comment]
turns out this was a mixup in package vs module naming.
We merged in #508 thanks to @hynek 
We'll make a new release soon so lancedb no longer brings `attr` in.

**Comment by changhiskhan** [comment]
fixed with #508
